StitchCore/StitchCore/Source/Core/Functions.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  Functions.swift
+//  StitchCore
+//
+//  Created by Jason Flax on 11/14/17.
+//  Copyright © 2017 MongoDB. All rights reserved.
+//
+
+import ExtendedJson
+
+internal struct Function: Encodable {
```

Comment:
[nit] I don't think this name conveys what this `struct` is. In the server we call this a `FunctionCallRequest`